include/morphio/properties.h

```diff
@@ -151,6 +151,8 @@ struct Marker {
 };
 
 struct CellLevel {
+    CellLevel()
+        : _version(MORPHOLOGY_VERSION_UNDEFINED) {}
     morphio::CellFamily _cellFamily;
     SomaType _somaType;
     MorphologyVersion _version;
```

Comment:
minor : _version being part of the initialization list now you can move it to the top of the member declaration.